openshift/system/config/backend.yml

```diff
@@ -1,4 +1,4 @@
 production:
   url: <%= ENV.fetch('BACKEND_URL', 'http://backend-listener:3000') %>
-  public_url: <%= ENV.fetch('BACKEND_ROUTE') { "https://backend-#{ENV['TENANT_NAME']}.#{ENV['THREESCALE_SUPERDOMAIN']}" } %>
```

Comment:
Route was used I guess because of openshift terms. To make clear it should be public. But I think the variable name you chose is clearer.

Reply:
Well, actually this is one of the sources of confusion :upside_down_face: 

Indeed, the name suggests that the value should be a publicly accessible URL, the value of OpenShift Route.

However, 3scale-operator was setting this env var to `http://backend-listener:3000/internal` (I pointed this out in the description of [this operator PR](https://github.com/3scale/3scale-operator/pull/821)).

Which also was wrong, and hence was causing https://issues.redhat.com/browse/THREESCALE-9558